src/minisat/mtl/Vec.h

```diff
@@ -42,8 +42,8 @@ class vec {
     int cap;
 
     // Don't allow copying (error prone):
-    vec<T>&  operator = (vec<T>& other) { assert(0); return *this; }
-             vec        (vec<T>& other) { assert(0); }
+    vec<T>&  operator = (vec<T>& other)  = delete;
```

Comment:
Align with the next line :) (Extra space?)

Reply:
Done :)